CMakeLists.txt

```diff
@@ -517,6 +517,10 @@ file(GLOB sources_instance_m_sc scripts/maelstrom/stonecore/*.cpp scripts/maelst
     LIST(APPEND mangosscript_LIB_SRCS ${sources_instance_m_sc})
     source_group("Eastern Kingdoms Scripts\\Instances\\Stonecore" FILES ${sources_instance_m_sc})
 endif()
+#Raid: Baradin Hold
```

Comment:
@i-am-fyre These lines should be above the endif() as where they are currently they will be picked up by all expansions

Reply:
Fixed.